src/components/home/Event.jsx

```diff
@@ -1,5 +1,32 @@
-const Event = ({ start, end, location, name, descripton }) => {
-  return <div>Event</div>;
+import { COLORS } from "@/data/colors";
+
+const Event = ({ start, end, location, name, description, color }) => {
+  const startDate = start.split("/");
+
+  return (
+    <div className=" bg-ewb-blue-100 w-1/2 inline-flex">
+      <div
+        className={`text-white text-center max-w-fit w-48 pt-6 pb-1 ${COLORS[color].bg}`}
```

Comment:
since you have w-3/4 in the component at right, you can do w-1/4 on this one to make sure that i takes 100%of the parent width, and you can  add an aspect-square to make this div a square. I dont think you need max-w-fit, and insted of pb-1 you can probably do p-1